lib/src/ui/movies_page.dart

```diff
@@ -14,11 +18,225 @@ class _MoviesPageState extends State<MoviesPage> {
+                    right: UiConstants.searchBarSidesPadding),
+                child: TextField(
+                  style: TextStyle(
+                      color: Colors.grey,
+                      fontSize: UiConstants.searchTextFontSize,
+                      fontFamily: 'Play-Bold'),
+                  decoration: InputDecoration(
+                      hintText: "Search for a movie...",
+                      hintStyle: const TextStyle(
+                        color: Colors.grey,
+                        fontWeight: FontWeight.w600,
+                      ),
+                      suffixIcon: const Icon(
+                        Icons.search_rounded,
+                        color: Colors.purple,
+                        size: UiConstants.searchIconSize,
+                      ),
+                      contentPadding: const EdgeInsets.only(
+                          left: UiConstants.searchTextPadding,
+                          right: UiConstants.searchTextPadding,
+                          top: UiConstants.searchTextPadding),
+                      filled: true,
+                      fillColor: Colors.white12),
+                ),
+              ),
+            ],
+          ),
+        ),
+      ),
+      body: SingleChildScrollView(
+        child: Padding(
+          padding: const EdgeInsets.all(UiConstants.bodyPadding),
+          child: Column(
+            children: <Widget>[
+              Align(
+                alignment: Alignment.centerLeft,
+                child: Padding(
+                  padding: const EdgeInsets.only(
+                      bottom: UiConstants.movieTypeTextPadding),
+                  child: new Text(
+                    'Trending Movies',
+                    style: TextStyle(
+                        fontSize: UiConstants.movieTypeTitleFontSize,
+                        color: Colors.grey,
+                        fontFamily: 'Play-Bold'),
+                  ),
+                ),
+              ),
+              StreamBuilder(
+                stream: _moviesBloc.swiper,
+                builder: (
+                  context,
+                  AsyncSnapshot<Movies> snapshot,
+                ) {
+                  return snapshot.hasData
+                      ? _swiper(
+                          snapshot,
+                        )
+                      : Center(
+                          child: Padding(
+                            padding: const EdgeInsets.only(
+                                top: UiConstants
+                                    .swiperProgressIndicatorTopPadding),
+                            child: CircularProgressIndicator(
+                              strokeWidth:
+                                  UiConstants.progressIndicatorStrokeWidth,
+                              backgroundColor: Colors.purple,
+                              valueColor: new AlwaysStoppedAnimation<Color>(
+                                  Colors.black),
```

Comment:
the "new" identifier is not necessary, and missed a trailing comma on the second line

Reply:
removed